src/components/tx/ExecutionMethodSelector/index.tsx

```diff
@@ -73,7 +72,7 @@ export const ExecutionMethodSelector = ({
         </FormControl>
       </div>
 
-      {shouldRelay && relays ? <SponsoredBy relays={relays} tooltip={tooltip} /> : wallet ? <BalanceInfo /> : null}
+      {shouldRelay && relays ? <SponsoredBy relays={relays} tooltip={tooltip} /> : null}
```

Comment:
I feel like we should still show the wallet's balance. Then it is easier to understand why the balance is too low or by how much the balance is too low.

Reply:
I've rebased this branch to the relabeling branch and kept the wallet balance.